Jenkinsfile

```diff
@@ -12,23 +12,31 @@ everything runs on whatever is labeled as 'master'
 node ('master'){
    /* receives the payload, parses it and gets us the PR number, templates to process and arch */
    stage('payload_processing') {
-
-      //clone the osl-jenkins on master branch to use the latest version of scripts
-      git url: 'https://github.com/osuosl-cookbooks/osl-jenkins', branch: 'master'
-
-      //write payload to a file
-      writeFile file: "/tmp/packer_pipeline_job_build_$BUILD_NUMBER", text: "$params.payload"
-
       //what event triggered this build ?
       event = new JsonSlurper().parseText("${params.payload}")['action']
 
       println "This build was triggered by the $event event. Look at https://developer.github.com/v3/activity/events/types for more info."
 
-      if ( event != 'synchronize' || event != 'opened' || event != 'review_requested' ) {
+      if ( event != 'synchronize' || event != 'opened' || event != 'review_requested' || event != 'created' ) {
          currentBuild.result = 'ABORTED'
          error("Stopping because this build was not triggered on a PR's synchronize/opened/review_requested event which would have all the necessary information for making a build successful")
       }
 
+      if ( event == 'created' ){
+        comment = new JsonSlurper().parseText("${params.payload}")['comment']['body']
+        if ( comment != '!deploy' ){
+          currentBuild.result = 'ABORTED'
+          error("Stopping this build because the comment was not a recognized command.")
+        }
+      }
```

Comment:
I'm trying to understand what this is trying to avoid. It looks like when we create a new PR, you want to make sure it doesn't have ``!deploy`` in it?

Edit: I read below in the comment that when a comment is created it uses the "created" event. Can you please comment it here too?